code/modules/events/mice_migration.dm

```diff
@@ -8,22 +8,17 @@
 	var/maximum_mice = 15
 
 /datum/round_event/mice_migration/announce(fake)
-	var/cause = pick("space-winter", "budget-cuts", "Ragnarok",
-		"space being cold", "\[REDACTED\]", "climate change",
-		"bad luck")
 	var/plural = pick("a number of", "a horde of", "a pack of", "a swarm of",
-		"a whoop of", "not more than [maximum_mice]")
-	var/name = pick("rodents", "mice", "squeaking things",
-		"wire eating mammals", "\[REDACTED\]", "energy draining parasites")
-	var/movement = pick("migrated", "swarmed", "stampeded", "descended")
-	var/location = pick("maintenance tunnels", "maintenance areas",
-		"\[REDACTED\]", "place with all those juicy wires")
+		"not more than [maximum_mice]")
+	var/name = pick("rodents", "mice")
+	var/movement = pick("migrated", "swarmed", "descended")
+	var/location = pick("maintenance tunnels", "maintenance areas")
 	if(prob(50))
-		priority_announce("Due to [cause], [plural] [name] have [movement] \
-		into the [location].", "Migration Alert",
+		priority_announce("Bioscans indicate that [plural] [name] have [movement] \
+		into [location].", "Migration Alert",
```

Comment:
Need to add //Skyrat changes here as well.

Reply:
Done

Reply:
I see a skyrat change start, but not a skyrat change stop. :P

Reply:
print_command_report("Bioscans indicate that [plural] [name] have [movement] into [location].", "Rodent Migration") //Skyrat change end

Its in, its simpy not showing because I took over your suggestion directly by clicking that button